src/main/kotlin/com/faforever/userservice/web/CloudflareHmacController.kt

```diff
@@ -69,4 +75,15 @@ class CloudflareHmacController(
 
         return HmacAccess(accessUrl)
     }
+
+    @GET
+    @Path("/hmac/token")
```

Comment:
I don't like the path, because it doesn't describe the business case behind it. `/lobby/access` and `/replay/access` are very clear on that.

What about something like `/challenge/token` ?

Reply:
That works